### ISSUE TITLE
Attack description are clipped when running 1.13.8+ in 1024x768

### ISSUE THREAD

**Issue by Arcanister** [wesnoth/wesnoth#1994]
Steps to reproduce:

1. Start any scenario in debug mode (e.g. tutorial), when window size is 1024x768 (perhaps other commons size too)
2. Create a unit with three or more attacks and at least two specials, e.g. Ancient Lich
3. Hover cursor over the newly created unit
3. The last attack description won't fit, you will only see first two completely and half of line "9x5 shadow wave"

This problem won't arise in 1.12.6, there seems to be more than enough place for everything to fit

**Comment by Wedge009** [comment]
Relating to the side-bar, I think it's the nature of the design. #1167, while describing symptoms specific to non-English translations, shows a related issue.

There's also been font and UI developments since 1.12 so text appears larger in 1.13. What you've found here is a boundary case where the dimensions just don't work any more for a unit with an unusually lengthy description. You can still hover over the text in the side-bar to get the full description, though.

It looks like font size is reduced for 800x600, so this isn't an issue for either 1.12 or 1.13.

Notes for replicating the issue: debug mode isn't necessary or related to the bug, it's just an easy way to get the desired unit.

**Comment by Arcanister** [comment]
I can't get information about attack type (arcane) and special (magical) for shadow wave by hovering mouse cursor anywhere. Is there any way to get information about remaining attacks of specific unit if it's not yours and it is not same as its unit type attack list?

**Comment by Arcanister** [comment]
I have noticed that this issue is related to Wesnoth interface theme. I have noticed that there are at least two of them:

- In one of them "End Turn" button is simple rectangle and it is smaller, minimap controls are on left size of minimap. When this theme is active, this problem doesn't arise in 1024x768, but still exist for 800x600. 800x642 is about minimum resolution which is enough to fit Ancient Lich full attack description.

- Another theme has thick textured End Turn button and minimap controls on the bottom of minimap. This theme has bigger font for attack description and it's the one where clipping happens. 

There is already interface to select theme in preferences. Would it be hard to add an option to select them manually, disabling automatic switching between two? Currently they both are applied when "Default" theme is selected depending on window size.
